### PR TITLE
Add formatting support for worker interactions

### PR DESCRIPTION
**Pull request by NipunaMarcus**
## Purpose
This will add formatting support for worker interactions.

### CODE REVIEW

File: language-server/modules/langserver-core/src/test/resources/formatting/expected/expectedWorkerInteractions.bal [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import ballerina/io;
```
Comment (hemikak) [inline language-server/modules/langserver-core/src/test/resources/formatting/expected/expectedWorkerInteractions.bal]: Copyright header missing.

File: language-server/modules/langserver-core/src/test/resources/formatting/workerInteractions.bal [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import ballerina/io;
```
Comment (hemikak) [inline language-server/modules/langserver-core/src/test/resources/formatting/workerInteractions.bal]: Copyright header missing.

File: language-server/modules/langserver-core/src/test/resources/formatting/expected/expectedWorkerInteractions.bal (hunk 1 above)
Comment (NipunaMarcus) [reply language-server/modules/langserver-core/src/test/resources/formatting/expected/expectedWorkerInteractions.bal]: these are test files for formatting @hemika ... these files are used for java level testing for formatting functionality ...  is header necessary there?

File: language-server/modules/langserver-core/src/test/resources/formatting/workerInteractions.bal (hunk 2 above)
Comment (NipunaMarcus) [reply language-server/modules/langserver-core/src/test/resources/formatting/workerInteractions.bal]: these are test files for formatting @hemika ... these files are used for java level testing for formatting functionality ...  is header necessary there?

**Pull request merged**